far/FarEng.hlf.m4

```diff
@@ -2506,56 +2505,54 @@ $ #Settings dialog: panel#
+  #Show column titles#      Enables displaying ~file panel~@FilePanel@ column titles.
 
-  #Определять точки#        Различать обычные связи для папок
-  #монтирования диска#      (Junction) и точки монтирования дисков
-                          томов (Volume).
-                          ^<wrap>Эта опция существенно замедляет отображение каталога на медленных сетевых подключениях.
+  #Show status line#        Enables displaying file panel status line.
 
-  #Show total#              Enable display of total information data
-  #information#             at the bottom line of file panel.
+  #Detect volume#           Distiguishes between normal directory links
+  #mount points#            (Junctions) and volume mount points.
+                          This option significanty slows down displaying
+                          directories on slow network connections.
 
-  #Show free size#          Enable display of the current disk free
-                          size.
+  #Show files total#        Enables displaying the file totals
+  #information#             at the bottom line of the file panel.
 
-  #Show scrollbar#          Enable display of file and ~tree panel~@TreePanel@
-  #in Panels#               scrollbars.
+  #Show free size#          Enables displaying the current disk free space.
 
-  #Show background#         Enable display of the number of
-  #screens number#          ~background screens~@ScrSwitch@.
+  #Show scrollbar#          Enables displaying file and ~tree panel~@TreePanel@ scrollbars.
 
-  #Show sort mode#          Indicate current sort mode in the
-  #letter#                  upper left panel corner.
+  #Show background#         Enables displaying the number of ~background screens~@ScrSwitch@.
+  #screens number#
 
+  #Show sort mode#          Displays current sort mode in the upper
+  #letter#                  left-hand corner of the panel.
 
-  #Подсвечивать#            Внутренние разделители колонок рисовать
-  #разделители колонок#     цветом, выбранным для конкретного типа
-                          файлового объекта.
-                          Если опция снята, разделители будут 
-                          отрисованы цветом внешних рамок.
+  #Show ".." in#            Enables displaying the ".." item on root folder
+  #root folders#            lists. Pressing #Enter# on this item opens
+                          ~Change drive menu~@DriveDlg@.
 
-  #Удваивать глобальные#    Вместо одинарной будет рисоваться 
-  #разделители колонок#     двойная внутренняя рамка между колонками.
+  #Highlight column#        If this option is on, internal column separators
+  #separators#              are displayed in the list item's color.
+                          Otherwise, the color of the outer frames is used.
 
+  #Double global#           If this option is on, the double internal frame
+  #column separators#       between columns is displayed instead of the single
+                          frame.
 
```

Comment:
What does this option do? I did not see any changes when I toggled this option in the build 5000.

Reply:
It applies to "primary" columns only (i.e. N).
If you set your panel mode to something like `N,S,N,S,N,S` you will see the difference:

![diff](https://user-images.githubusercontent.com/11453922/35793101-27d55864-0a48-11e8-9eb0-d6597e68c78f.png)

It was introduced to make it easier to recognise global primary (`N`) and non-primary columns (e.g. `S`) when `Highlight column separators` is not enabled.

The image above also demostrates what does `Highlight column separators` do.

Reply:
Thank you. I'll update description of `Double global column separators` in the next PR.